### PR TITLE
Add a link to symbol/function support preview page

### PR DESCRIPTION
**Pull request by utensil**
As suggested by @kevinbarabash in https://github.com/Khan/KaTeX/issues/103#issuecomment-148118466 , I've added a link to a symbol/function support preview page that I automatically generated and handcrafted in CONTRIBUTING.md.

This PR is not meant to be directly merged(though that's also possible), but opens a discussion about how to integrate this preview page into contributing guide.

The preview page currently lists all symbols and function in KaTeX wiki and MathJax document, and attempts to render them, and falls into 3 kind of results:

* successfully rendered by just the symbol itself, presented in a green cell
* can't be rendered by itself(because it's function), but can be successfully rendered by providing an proper example,  presented in a blue cell, along with the tooltip of the original TeX code of the example, that's the main handcraft part
* failed to render  (because KaTeX doesn't support them in the release yet), presented in a red cell

The page is quite self-explanatory and also includes statistics of the three kind of results.

The question left to be discussed are:

* is the preview page proper/good enough to be added to CONTRIBUTING.md ?
* should the preivew page be somewhere more official, like in a rep of KaTeX organization? (the code is not necessary good enough, because it's just a prototype, though it works nicely)
* should the introduction of the 3 places that contributors to check for symbol/function support improve wording and organization?

P.S. I've already signed the SLA.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,10 +10,13 @@ If you'd like to contribute, try contributing new symbols or functions that
...
 functions](https://github.com/Khan/KaTeX/wiki/Function-Support-in-KaTeX) as 
-well as a page that describes how to [examine TeX commands and where to find
-rules](https://github.com/Khan/KaTeX/wiki/Examining-TeX) which can be quite
-useful when adding new commands. You
-can check there to see if we don't support a function you like, or try your
+well as a page that describes how to [examine TeX commands and where to find 
+rules](https://github.com/Khan/KaTeX/wiki/Examining-TeX) which can be quite 
+useful when adding new commands. There's also a user-contributed [preview page]
+(http://utensil-site.github.io/available-in-katex/)
+showing how would KaTex render a series of symbols/functions(including the ones
```
Comment (kevinbarabash) [inline CONTRIBUTING.md]: "showing how KaTeX would"... and "symbols/functions (including"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,10 +10,13 @@ If you'd like to contribute, try contributing new symbols or functions that
...
+MathJax listed in their documentation and the extra ones supported by KaTeX). You
+can check them to see if we don't support a function you like, or try your
```
Comment (kevinbarabash) [inline CONTRIBUTING.md]: "we don't support" => "we support"

**Comment by kevinbarabash** [comment]
@utensil use the katex.min.js and katex.min.css served off of khan.github.io is an interesting way to stay current.   I think this should be okay unless we change were these files are located which seems unlikely, but it is a potential failure point.  I think having the code for this in your repo is fine for the time being.  If there are changes we want to make we'll fork it at that point.

One thought about the layout... instead of having three colors, it might be better to have three columns: one for the command, one for the code use to generate the rendering, and then the rendering itself (or a a failure message like you have now).

Thanks for signing the CLA.

File: CONTRIBUTING.md (hunk 1 above)
Comment (utensil) [inline CONTRIBUTING.md]: I get the "showing how KaTeX would" part, but what does "symbols/functions (including" part mean? I've checked it letter by letter, failed to find the difference.

File: CONTRIBUTING.md (hunk 2 above)
Comment (utensil) [inline CONTRIBUTING.md]: This is the original wording, unchanged. I guess it means that the contributor would look for what's missing and add them. Should I change the meaning to the other way around(the users looking for what's already at hand)?

**Comment by utensil** [comment]
> @utensil use the katex.min.js and katex.min.css served off of khan.github.io is an interesting way to stay current. I think this should be okay unless we change were these files are located which seems unlikely, but it is a potential failure point.

I totally agree. I started with the CDN version in README.md, but it seems to be not loading as fast as the github.io version, so I went with it for the time being.

> I think having the code for this in your repo is fine for the time being. If there are changes we want to make we'll fork it at that point.

OK, that's point of open source and Github, no central point of failure, :wink:  And if it becomes somehow official later, there would always be a better way to stay current.

> One thought about the layout... instead of having three colors, it might be better to have three columns: one for the command, one for the code use to generate the rendering, and then the rendering itself (or a a failure message like you have now).

Yeah. The thought crossed my mind too. I didn't use it because it's a little verbose for the symbols and I planed to do something similar when I find a proper way to separate symbols and functions(and meta characters).

File: CONTRIBUTING.md (hunk 2 above)
Comment (kevinbarabash) [inline CONTRIBUTING.md]: Sorry, I didn't look at the original wording.  We can leave it as is.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kevinbarabash) [inline CONTRIBUTING.md]: There should be a space between before the "(".

Comment (utensil) [inline CONTRIBUTING.md]: Oh I see.

**Comment by utensil** [comment]
Fixed. Should I squash my commits in this branch into one and `git push --force`? Not quite did that before.

And travis-ci got this weird error: `can't resolve github.com`, seems to be temporary.

**Comment by kevinbarabash** [comment]
> Should I squash my commits in this branch into one and `git push --force`?

Yes please.

**Comment by utensil** [comment]
Is it OK now?

**Comment by kevinbarabash** [comment]
@utensil looks good.  Thanks for the pull request.

**Pull request merged**